component/src/main/wso2/kafka/consumer.bal

```diff
@@ -16,43 +16,44 @@
-# + topics - topics to be subscribed
-# + properties - additional properties if required
-# + sessionTimeout - timeout used to detect consumer failures when heartbeat threshold is reached
-# + heartBeatInterval - expected time between heartbeats
-# + metadataMaxAge - max time to force a refresh of metadata
-# + autoCommitInterval - offset committing interval
-# + maxPartitionFetchBytes - the max amount of data per-partition the server return
-# + sendBuffer - size of the TCP send buffer (SO_SNDBUF)
-# + receiveBuffer - size of the TCP receive buffer (SO_RCVBUF)
-# + fetchMinBytes - minimum amount of data the server should return for a fetch request
-# + fetchMaxBytes - maximum amount of data the server should return for a fetch request
-# + fetchMaxWait - maximum amount of time the server will block before answering the fetch request
-# + reconnectBackoffMax - maximum amount of time in milliseconds to wait when reconnecting
-# + retryBackoff - time to wait before attempting to retry a failed request
-# + metricsSampleWindow - window of time a metrics sample is computed over
-# + metricsNumSamples - number of samples maintained to compute metrics
-# + requestTimeout - wait time for response of a request
-# + connectionMaxIdle - close idle connections after the number of milliseconds
-# + maxPollRecords - maximum number of records returned in a single call to poll
-# + maxPollInterval - maximum delay between invocations of poll
-# + reconnectBackoff - time to wait before attempting to reconnect
-# + pollingTimeout - time out interval of polling
-# + pollingInterval - polling interval
-# + concurrentConsumers - number of concurrent consumers
-# + autoCommit - enables auto commit offsets
-# + checkCRCS - check the CRC32 of the records consumed
-# + excludeInternalTopics - whether records from internal topics should be exposed to the consumer
-# + decoupleProcessing - decouples processing
+# + bootstrapServers - List of remote server endpoints
+# + groupId - Unique string that identifies the consumer
+# + offsetReset - Offset reset strategy if no initial offset
+# + partitionAssignmentStrategy - Strategy class for handle partition assignment among consumers
+# + metricsRecordingLevel - Metrics recording level
+# + metricsReporterClasses - Metrics reporter classes
+# + clientId - Id to be used for server side logging
+# + interceptorClasses - Interceptor classes to be used before sending records
+# + isolationLevel - How the transactional messages are read
+# + topics - Topics to be subscribed
+# + properties - Additional properties if required
+# + sessionTimeout - Timeout used to detect consumer failures when heartbeat threshold is reached
+# + heartBeatInterval - Expected time between heartbeats
+# + metadataMaxAge - Max time to force a refresh of metadata
+# + autoCommitInterval - Offset committing interval
+# + maxPartitionFetchBytes - The max amount of data per-partition the server return
```

Comment:
return -> returns

Reply:
Also, shall we consistently use either max or maximum (better if maximum is used)? In some places it says maximum.